samples/busted_responses/bmo_v102__no_header_newline.qfx

```diff
@@ -0,0 +1,59 @@
+OFXHEADER:100
+DATA:OFXSGML
+VERSION:102
+SECURITY:NONE
+ENCODING:USASCII
+CHARSET:1252
+COMPRESSION:NONE
+OLDFILEUID:NONE
+NEWFILEUID:NONE
+<OFX>
+<SIGNONMSGSRSV1>
+<SONRS>
+<STATUS>
+<CODE>0
+<SEVERITY>INFO
+<MESSAGE>OK
+</STATUS>
+<DTSERVER>20181202184906.217[-5:EDT]
+<USERKEY>SJLDF802DV09DF80
+<LANGUAGE>ENG
+<INTU.BID>00017
```

Comment:
FYI, I think this `<INTU.BID>` tag is the only one in here that makes this QFX instead of OFX.

Reply:
Interesting. Good news, it doesn't seem to cause any trouble parsing the file. I assume it's just ignored, but I don't need it.

Reply:
Yup, exactly. Go's XML parsing just ignores extraneous tags, so it's easy for us to accept QFX without complaint... As long as it doesn't also remove random newlines :stuck_out_tongue: